packages/filebrowser/src/listing.ts

```diff
@@ -2493,6 +2494,7 @@ export namespace DirListing {
      */
     createCheckboxWrapperNode(options?: {
       alwaysVisible: boolean;
+      headerNode: boolean
```

Comment:
If we add `?` here this will allow us to backport it (as otherwise the new argument changes API in a breaking way).

```suggestion
      headerNode?: boolean
```